xml/obs_concepts_scm.xml

```diff
@@ -100,7 +100,7 @@
    local archive. </para>
  </sect2>
  <sect2>
-  <title>Manage your build description in SCM</title>
+  <title>Manage Your Build Description in a SCM</title>
```

Comment:
Is build description the same than (build) recipe? If not, shall we add it to the glossary?